app/src/main/java/com/android/shelfLife/ui/newProfile/ProfileScreen.kt

```diff
@@ -146,7 +145,10 @@ fun ProfileScreen(
 
               // Logout button
               OutlinedButton(
-                  onClick = { profileViewModel.signOut(context) },
+                  onClick = {
+                    profileViewModel.signOut(context)
+                    navigationActions.navigateToAndClearBackStack(Screen.AUTH)
```

Comment:
I think its a good aproach to use `navigateToAndClearBackStack` in order to make sure that the users' history is also erased